PicoTags.php

```diff
@@ -75,6 +88,21 @@ public function onPagesLoaded(&$pages, &$currentPage, &$previousPage, &$nextPage
         }
     }
 
+    /**
+     * Register Twig's tags_all.
+     * I don't know if it has to be in this method; just copying from http-params ;)
+     *
+     * @see DummyPlugin::onPageRendering()
+     */
+    public function onPageRendering(Twig_Environment &$twig, array &$twigVariables, &$templateName)
+    {
+        $twig->addFunction(new Twig_SimpleFunction('tags_all', array($this, 'getTags')));
```

Comment:
How about naming the twig function `get_all_tags` instead (and the method `getAllTags`) to make it a bit more descriptive and consistent?